### DISCUSSION TITLE
How get access to the .NET 6 version

### DISCUSSION THREAD

**Discussion by BorisGoryunov** [DanWBR/dwsim#537] [General]
In this topic https://github.com/DanWBR/dwsim/issues/317
you mention a repository with .net 6 version
How could I get access to the repository ?

**Comment by DanWBR** [comment]
@BorisGoryunov may I ask what is your purpose, Boris? the repository is not up-to-date.

**Reply by BorisGoryunov** [reply]
Hello! I am developing on .net6 C# for Linux. How can I use your repository for development? Is this possible?
You wrote here https://github.com/DanWBR/dwsim/issues/317 that there is a repository for .net6. Can I access it? Or is it not up to date?
